New mission Storage Wars

CassYoko01 opened the pull request:
Mission Pull Request Template

This Pull Request will:

Add the mission Storage Wars made by CassYoko to the repository.

**Mission Creation Checklist:**
- [x] Mission File
- [x] Gearscripts Tested
- [x] Ingame Play Area Markers
- [x] Ingame Briefings to all players
- [x] Side-based briefings  
- [x] Mission .conf file


**Faction(s):**

BLUFOR - UK 80s - Defending
BLUFOR Comp: Infantry Platoon
BLUFOR Assets: 1x MMG Team

OPFOR - USSR 80s - Attacking
OPFOR Comp: Motorized Company
OPFOR Assets:  2x BDRM2

**Objectives/Win Conditions:**

BLUFOR: Defend both bomb sites.

OPFOR: Destroy both bomb sites

**Terrain:**

Bearland

**Short mission description:**

USSR forces are being deployed to destroy the docks and a nearby storage facility.  UK forces have become aware of the attack and established defensive positions around both targets.

**Slotting Ratio:**

3:2 (OPFOR:BLUFOR)

**Time limit:**

45 min

**Mission Briefing Screenshot:**

<img width="826" height="660" alt="Screenshot 2025-08-31 215706" src="https://github.com/user-attachments/assets/5814b718-34b7-4fbd-9b7c-45e36c4202ca" />

**Design concept/thoughts:**

New map, was going to make this a rush mission.

**Other Notes:**

I encountered the game boundary bug, where, on the map, it displays out-of-bounds areas throughout the entire map. In-game, the game boundary still works, just a visual bug.

RandAlThor1 commented:
**QA Feedback**
- callsigns for BLUFOR are wrong. extra callsigns will rollover and break things. need to make sure there is the same amount of names as there is groups
<img width="359" height="359" alt="image" src="https://github.com/user-attachments/assets/ecfeea5e-2fcc-4b9a-8746-9649e8a2143c" />
<img width="651" height="863" alt="image" src="https://github.com/user-attachments/assets/e1290831-416a-439c-b391-00f40af9a61a" />

- it can be very finicky  but if you mess around with the game border you should be able to solve the issue with the inside being shaded. also reall this map doesn't really need one since its already so small.

- very minor but no space here in the mission config
<img width="150" height="87" alt="image" src="https://github.com/user-attachments/assets/521f0cf9-112b-45b8-afc1-c1af3e4d47db" />

CassYoko01 commented:
Removed Game boundary
Fixed BLUFOR slotting

New Mission image to reflect changes
<img width="941" height="705" alt="Screenshot 2025-09-03 142250" src="https://github.com/user-attachments/assets/c2364124-ebbe-4a42-9d3a-bac76709ee67" />
